definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-alt-service-process-server-nonprod.json

```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "LiveFrom": "17/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "servedByProcessServer",
```

Comment:
Field names shouldn't start with uppercase "S"?

Reply:
Good point. I'll change that now.

Reply:
Done